game/match.py

```diff
@@ -41,5 +38,14 @@ def who_won(self):
                 return player
         return None
 
+    def retrieve_board(self):
+        return copy(self._board)
```

Comment:
`GameBoard` already implements this https://github.com/apojomovsky/cuatro_en_linea/blob/master/game/gameboard.py#L74